### ISSUE TITLE
White space on right and bottom of page on wkhtmltopdf 0.12.6 latest version

### ISSUE THREAD

**Issue by samiramaspl** [wkhtmltopdf/wkhtmltopdf#5088]
**wkhtmltopdf version(s) affected**: 0.12.6

**Description**
I have facing same issue which many people faces on various earlier versions also but still not fixed by wkhtmltopdf
I get white space on right and bottom of my page after generating PDF, so when it will be resolved?

**How to reproduce**
sample. html
```
<!DOCTYPE html>
<html>
<head>
</head>
<body style="margin: 0; padding: 0;">
<div style="height: 30mm; width: 45mm; background-color: #F7EBD4;">
&nbsp;
</div>
</body>
</html>
```
Command to generate PDF from HTML

```
wkhtmltopdf -L 0 -R 0 -T 0 -B 0 --page-width 45mm --page-height 30mm --disable-smart-shrinking --enable-local-file-access sample.html sample.pdf
```
Generated PDF 
[sample.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/7189378/sample.pdf)

**Expected behavior**
I want to remove white space from right and bottom of the generated PDF, how can be it fix?

**Comment by jaydevvaghela943** [comment]
I have same issue..............

**Comment by XiaoFeiHe1993** [comment]
I also have，You can see this white only when you enlarge pdf with 300%。

**Comment by samiramaspl** [comment]
Yes, it seems it is pending to resolve since long time but it seems not in development roadmap, So I finally I have switched to paid tool in less price - https://typeset.sh/en So who have problem can try this
